salt/utils/schedule.py

```diff
@@ -474,6 +477,7 @@ def run_job(self, name):
         # Grab run, assume True
         run = data.get('run', True)
         if run:
```

Comment:
Just check `data['run']` as 10 lines up we guarantee data['run'] exists?

Reply:
:+1: Good catch, definitely more efficient.